python/tank/descriptor/io_descriptor/appstore.py

```diff
@@ -24,7 +24,6 @@
 from ...util import UnresolvableCoreConfigurationError, ShotgunAttachmentDownloadError
 from ...util.user_settings import UserSettings
 
-from ..descriptor import Descriptor
```

Comment:
Could you explain why we do not import this globally class anymore? I'm worried about the local import that I found below in the `_register_enums` method.

Reply:
So the immediate reason to do this was to avoid a cyclic import that's created from 
[python/tank/descriptor/descriptor.py:16](https://github.com/shotgunsoftware/tk-core/blob/SG-9806-github_release_descriptor/python/tank/descriptor/descriptor.py#L16).  Is there a better way to deal with this?

Reply:
JF and I took a look at this together this morning and he proposed a good solution -- move the Enum that's currently defined on Descriptor to constants.